build/bin/write-dockerfile.sh

```diff
@@ -55,7 +55,10 @@ case $SYSTEM in
                 ;;
             *)
                 cat <<EOF
-RUN (yes | unminimize) || echo "(ignored)"
+RUN if command -v unminimize > /dev/null; then  \
+        (yes | unminimize) || echo "(ignored)"; \
+        rm -f "$(command -v unminimize)";       \
+    fi
 EOF
```

Comment:
What is `unminimize`? Why is it removed?

Reply:
The Ubuntu Docker images are "minimized", meaning that some large bits such as documentation has been removed. We have to unminimize once (which reinstalls the full versions of some minimized packages), or e.g. the maxima documentation (which we depend on for correct operation) will be missing (IIRC). But we only have to do this once. To save time in incremental builds, I'm removing the `unminimize` binary here.

Reply:
OK. 

Generally speaking, I hope that any cryptic part of a script is annotated with ample comments like the one you just gave, to ease the work of the future maintainer of our CI infrastructure.

Reply:
Great suggestion, done in 17583ab633a and c18ba755eba